packages/react-components/src/organisms/RecentSharedOutputs.tsx

```diff
@@ -118,9 +118,11 @@ const RecentSharedOutputs: React.FC<RecentSharedOutputProp> = ({ outputs }) => (
               {getIconForDocumentType(documentType)} {documentType}
             </p>
             <span css={[titleStyles, rowTitleStyles]}>Date</span>
-            <span>
-              {formatDateToTimezone(addedDate, 'E, d MMM y').toUpperCase()}
-            </span>
+            {addedDate && (
```

Comment:
Here I know that all the "Recent Shared outputs" are Published ones that we are displaying on the dashboard, and with the new type I could avoid doing this check.